src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -570,13 +576,13 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
     this.setResolutionStepValidationDetail(validationDetail)
   }
 
-  private onResolutionConfirmedChange (resolutionConfirmed: boolean): void {
-    this.hasResolutionConfirmed = resolutionConfirmed
-    this.updateResolutionStepValidationDetail()
+  private async onResolutionConfirmedChange (resolutionConfirmed: boolean): Promise<void> {
+    await Vue.nextTick()
```

Comment:
Please add a comment why this is needed.

Reply:
added